source/includes/_chat_summary.md

```diff
@@ -0,0 +1,174 @@
+## Chat Summary
+
+### Resource Overview
+
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/chat_summary/[gmaid]?[query_params]|
+
+### API Name: Chat Summary
+
+### Summary
+
+This API can be used by customers who have our chat widget on their site. The API has metrics on the chat along with traffic information (Paid Campaign/Organic).
```

Comment:
Something not quite right with second sentence.  Should it be "each chat" or "chats" perhaps?

Reply:
This has been updated to "each chat".